ansible/roles_ocp_workloads/ocp4_workload_performance_monitoring/files/che-cluster.yml

```diff
@@ -0,0 +1,47 @@
+apiVersion: org.eclipse.che/v2
+kind: CheCluster
+metadata:
+  name: devspaces
+  namespace: openshift-devspaces
+spec:
+  devEnvironments:
+    defaultEditor: che-incubator/che-code/insiders
+    defaultNamespace:
+      template: <username>-devspaces
```

Comment:
CheCluster custom resource understands the <> notation?  I'm asking out of my ignorance, accustomed to using jinja.

Reply:
nice